config/packages/test/security.yaml

```diff
@@ -1,12 +1,12 @@
 security:
     providers:
-        testprovider:
+        sulu:
             id: test_user_provider
 
     firewalls:
         admin:
             http_basic:
-                provider: testprovider
+                provider: sulu
```

Comment:
This fixes:

```
  Not configuring explicitly the provider for the "form_login" listener on "s  
  ulu_test" firewall is ambiguous as there is more than one registered provid  
  er.     
```

In 1.6 this was needed as the test user provider did only handle the test user but in 2.x now thes testuserprovider does fallback now to the sulu user provider so we can just override here in the test env that the `test_user_provider` can be used.